ISISComputingGroup/ibex_gui
# Added ability to individually remove users from the experiment

###Description of work

Fixes https://github.com/ISISComputingGroup/IBEX/issues/1322

###To test

* Add a few new users on the experiment details perspective
* Remove one, confirm that it is removed from this list as well as the dashboard

See system test at https://github.com/ISISComputingGroup/ibex_system_tests/pull/15

---

#### Code Review


### Functional Tests
